Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -468,6 +505,137 @@ def block_stabilizerOrderIso [htGX : IsPretransitive G X] (a : X) :
 
 end Stabilizer
 
+section Finite
+
+namespace IsBlock
+
+variable [IsPretransitive G X] [Finite X] {B : Set X} (hB : IsBlock G B)
```

Comment:
The first three theorems do not require `[Finite X]`.

Reply:
Good point. I deleted the hypothesis and only add it for the lemmas that need it.